rest_framework/relations.py

```diff
@@ -71,8 +72,8 @@ def __str__(self):
 
 class RelatedField(Field):
     queryset = None
-    html_cutoff = 1000
-    html_cutoff_text = _('More than {count} items...')
+    html_cutoff = int(api_settings.HTML_SELECT_CUTOFF)
+    html_cutoff_text = _(api_settings.HTML_SELECT_CUTOFF_TEXT)
```

Comment:
Might need to be something like:

Don't include these attribute on the class unless overridden, then in `__init__`, do something like...

    if `html_cutoff` in kwargs:
        self.html_cutoff = kwargs.pop('html_cutoff')
    elif not hasattr(self, 'html_cutoff`):
        # Use default from settings, unless already set as a class attribute.
        self.html_cutoff = int(api_settings.HTML_SELECT_CUTOFF)